src/OMSimulatorLib/Model.cpp

```diff
@@ -264,6 +288,36 @@ oms_status_enu_t oms::Model::list(const oms::ComRef& cref, char** contents)
   return oms_status_ok;
 }
 
+/*
+ * This function update the ParameterBindings in SSD, to link with a SSV file, (e.g)
+ * <ssd:ParameterBindings>
+ *     <ssd:ParameterBinding source="resources/import_export_parameters.ssv" />
+ * </ssd:ParameterBindings>
+ */
+oms_status_enu_t oms::Model::updateParameterBindingsToSSD(pugi::xml_node& node, pugi::xml_node& ssvNode, bool isTopSystemOrModel) const
+{
+  int parameterNodeCount = std::distance(ssvNode.begin(), ssvNode.end());
+
+  // check parameter bindings exist and export to ssv file and also update the ssd file with parameterBindings at the top level
+  if (parameterNodeCount > 0 && isTopSystemOrModel)
+  {
+    // update the ssd with the top level parameterBindings (e.g)  <ParameterBinding source="resources/ControlledTemperature.ssv">
+    for(pugi::xml_node_iterator it = node.begin(); it != node.end(); ++it)
+    {
+      if (std::string(it->name()) == oms::ssp::Draft20180219::ssd::connectors) // insert the parameter bindings after top-level connectors node
+      {
+        pugi::xml_node node_parameters_bindings = node.insert_child_after(oms::ssp::Version1_0::ssd::parameter_bindings, *it);
+        pugi::xml_node node_parameter_binding  = node_parameters_bindings.append_child(oms::ssp::Version1_0::ssd::parameter_binding);
+        std::string ssvFileName = std::string(this->getCref()) + ".ssv";
```

Comment:
Shouldn't it be `resources/<name>.ssv`?

Reply:
This is correct , now the ssv file is exported to resource directory